packages/discount-code-exporter/src/main.js

```diff
@@ -164,12 +157,15 @@ export default class DiscountCodeExport {
       'customLineItemFieldTypes',
     ]
     objKeys.forEach((key: string) => {
-      if (restCodeData[key] && !Object.keys(restCodeData[key]).length)
-        delete restCodeData[key]
+      if (
+        restDiscountCodeData[key] &&
+        !Object.keys(restDiscountCodeData[key]).length
+      )
+        delete restDiscountCodeData[key]
```

Comment:
nooo I didn't see this earlier 😭  #mutation 
wouldn't it be cooler to use `entries` + `reduce` here?

at the top of my head: 

```
const objKeys = [
  'attributeTypes',	      
  'cartFieldTypes',	       
  'lineItemFieldTypes',	     
  'customLineItemFieldTypes',
]

discountCodeData = Object.entries(restDiscountCodeData).reduce(
  (discountCode, ([discountCodeKey, value])) =>
    objKeys.includes(discountCodeKey)
      ? {
        ...discountCode,
        [discountCodeKey]: undefined
      } : discountCode,
  {}
)
```